### PR TITLE
Edit job

### PR DESCRIPTION
**Pull request by lukasz-kaniowski**
* [x] new edit job page where user can update `progress` `rating` 
* [x] move delete button from dashboard to edit job page
* [x] add update button to dashboard page
* [x] move update `progress` from dashboard page to edit job page
* [x] add header copy to `add job` page

### CODE REVIEW

File: app/controllers/update-job-view-model.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const i18n = require('i18n');
+const progression = require('../models/progression');
+/* eslint-disable no-underscore-dangle */
+
+class UpdateJobViewModel {
+  constructor(accountId, job) {
+    Object.assign(this, job);
+    this.accountId = accountId;
+
+    const ratingOptionList = [5, 4, 3, 2, 1].map(value => ({ value }));
```
Comment (nathancashmoredwp) [inline app/controllers/update-job-view-model.js]: Maybe worth carrying out a quick manual check to make sure the right ratings end up in the DB.

File: app/controllers/jobs-controller.js [hunk 2]
```diff
@@ -84,8 +95,8 @@ router.patch('/:jobId', (req, res, next) => {
   return new Jobs({ id: jobId })
     .save(updateData, { method: 'update', patch: true })
     .then((job) => {
-      const queryParams = `sort=${sort}&filter=${filter}&focus=${job.id}#${job.id}`;
-      res.redirect(`${basePath}/${accountId}?${queryParams}`);
+      const queryParams = buildQueryParams(req, job);
```
Comment (nathancashmoredwp) [inline app/controllers/jobs-controller.js]: Where did the focus parameter go... ? as i can't see it in the buildQueryParams

File: test/integration/update-job.spec.js [hunk 3]
```diff
@@ -60,6 +60,14 @@ describe('Update a job', () => {
     );
   });
 
+  describe('update job details', () => {
+    it('should update job progress', () =>
+      updateJobPage.setJobProgression('result')
+        .then(() => updateJobPage.submit())
+        .then(() => expect(helper.dashboardPage.getJobProgressionStatus(job)).to.equal('Result'))
```
Comment (nathancashmoredwp) [inline test/integration/update-job.spec.js]: Does this test need to cycle through all the possible job progression states ?

File: test/integration/update-job.spec.js [hunk 4]
```diff
@@ -66,6 +66,12 @@ describe('Update a job', () => {
         .then(() => updateJobPage.submit())
         .then(() => expect(helper.dashboardPage.getJobProgressionStatus(job)).to.equal('Result'))
     );
+
```
Comment (nathancashmoredwp) [inline test/integration/update-job.spec.js]: Should this check all of the possibilities ?

File: app/controllers/jobs-controller.js [hunk 5]
```diff
@@ -25,10 +25,8 @@ function validateDeadline(deadline, req) {
   }
 }
 
-function buildQueryParams(req, job) {
-  const sort = req.query.sort;
-  const filter = req.query.filter;
-  const params = { sort, filter, focus: job.id };
+function buildQueryParams(job) {
+  const params = { focus: job.id };
```
Comment (nathancashmoredwp) [inline app/controllers/jobs-controller.js]: Forget the focus id comment...

File: app/controllers/update-job-view-model.js (hunk 1 above)
Comment (lukasz-kaniowski) [inline app/controllers/update-job-view-model.js]: I had a look. all good

File: test/integration/update-job.spec.js (hunk 3 above)
Comment (lukasz-kaniowski) [inline test/integration/update-job.spec.js]: check the latest commit

**Pull request merged**